src/serve.js

```diff
@@ -6,20 +6,23 @@ const colors = require('./colors');
 const checkInstall = require('./check-install');
 const exec = require('./exec');
 
-
 module.exports = {
   cues: ['serve', 's', '-s', '--serve', 'watch', 'w', '--watch'],
   hint: 'serve your docs on localhost for development.',
   run: async () => {
     await checkInstall();
     shell.echo(chalk`{${colors.success} #} Serving ...`);
     shell.echo(chalk`{${colors.success} #} Fetching project configuration ...`);
-    const core_installed = require(path.resolve('.codedoc/package-lock.json'))
-                          .dependencies['@codedoc/core'].version.split('.');
-    if (core_installed[0] < 1 && core_installed[1] < 2) {
-      await exec('ts-node', '-T --project .codedoc/tsconfig.json .codedoc/watch');
-    } else {
-      await exec('ts-node-dev', '-T --project .codedoc/tsconfig.json .codedoc/serve');
+    const core_installed = require(path.resolve('.codedoc/package-lock.json')).dependencies['@codedoc/core'].version.split('.');
+
+    try {
+      if (core_installed[0] < 1 && core_installed[1] < 2) {
+        await exec('ts-node', '-T --project .codedoc/tsconfig.json .codedoc/watch');
+      } else {
+        await exec('ts-node-dev', '-T --project .codedoc/tsconfig.json .codedoc/serve');
+      }
+    } catch (error) {
+      error && console.error('\nError !!!\n', error);
```

Comment:
I suspect it would be better if all error logging from the CLI was consistent. You can see another example of that [here](https://github.com/CONNECT-platform/codedoc-cli/blob/45ef4fdb14d849e72a431e2e8ac1add70dd7a774/src/build.js#L18-L22).

Reply:
done!